lighthouserc.json

```diff
@@ -0,0 +1,7 @@
+{
+  "ci": {
+    "collect": {
+      "staticDistDir": "./website/build"
```

Comment:
Instead of adding a staticDistDir. I believe this should be done 

"startServerCommmand" : "yarn start",
"url" : [The URL being output by Webpack]

Reply:
Do you know what is the "The URL being output by Webpack" ? I'm not sure about that.
Also `yarn start` will start the site on the dev server which I don't believe will give the same lighthouse scores as the actual build.